.tav.yml

```diff
@@ -221,6 +221,12 @@ restify:
   commands:
     - node test/instrumentation/modules/restify.js
 
+fastify:
+  versions: '>=0.27.0 <0.29.0 || >0.29.0'
```

Comment:
I would recommend to use Fastify ≥1, pre v1 versions where highly unstable and nobody should use them :)

Reply:
Yeah, maybe there's no need to test this. And indeed we might even bump it to v2+ as that's the only range where we also support error handling. But normally I just go as low as we can without any modifications to our code and it turned out that in this case the `onRequest` hook was supported in the current configuration all the way down to v0.27 - so that's why I added that as a minimum - i.e. we got it basically for free.